modules/debezium_connection/debezium-db-config.tpl

```diff
@@ -0,0 +1,49 @@
+{
+  "bootstrap.servers": "${bootstrap_servers}",
+  "acks": "all",
+
+  "database.dbname": "${database_name}",
+  "database.hostname": "${database_address}",
+  "database.password": "${database_password}",
+  "database.port": "5432",
+  "database.server.name": "${connection_name}",
+  "database.user": "${database_user}",
+  "heartbeat.action.query": "${heartbeat_query}",
+  "heartbeat.interval.ms": "30000",
+
+  "behavior.on.null.values": "delete",
+  "connector.class": "io.debezium.connector.postgresql.PostgresConnector",
+  "database.initial.statements": "DO $$BEGIN IF NOT EXISTS(SELECT FROM pg_publication WHERE pubname = 'debezium_public_events') THEN CREATE PUBLICATION debezium_public_events FOR TABLE ${events_table} WITH (publish = 'insert');; END IF;; END$$;",
```

Comment:
This should be removed.  Creating a publication doesn't work with the reduced privilege debezium user.